src/SonarScanner.MSBuild.PreProcessor/Roslyn/EmbeddedAnalyzerInstaller.cs

```diff
@@ -39,9 +39,9 @@ namespace SonarScanner.MSBuild.PreProcessor.Roslyn
     /// can't then use them.
     /// </para>
     /// <para>
-    /// The plugin resources are cached locally under %temp%\.sonarqube\.static\[package_version]\[resource]
-    /// If the required version is available locally then it will not be downloaded from the
-    /// SonarQube server.
+    /// The plugin resources are cached locally under %temp%\.sonarqube\resources\index.
```

Comment:
```suggestion
    /// The analyzer resources are cached locally under %temp%\.sonarqube\resources\ (each analyzer inside a subfolder).
```

- there is no `index` folder, might be confusing
- S4NET operates with analyzers rather than plugins